woocommerce/payment-gateway/integrations/class-sv-wc-payment-gateway-integration-subscriptions.php

```diff
@@ -661,6 +663,61 @@ public function add_my_payment_methods_table_header( $headers, $handler ) {
 	}
 
 
+	/**
+	 * Displays a list of subscriptions orders associated with the current token.
+	 *
+	 * This method only generates output for tokens created by the framework.
+	 *
+	 * TODO: Ask Subscriptions developers whether they would be interested in start adding the column to the payment methods table themselves {WV 2020-02-19}
+	 *
+	 * @since x.y.z
```

Comment:
I have been using 5.6.0-dev. I just want to make sure we are using the same...

Reply:
Makes sense. I should have checked. I fixed the two instances here and two other I found in 318c777